simcon_project/conversation_templates/models/template_response.py

```diff
@@ -14,13 +14,15 @@ class TemplateResponse(models.Model):
     template: ConversationTemplate object that TemplateResponse relates to
     assignment: Assignment object that ConversationTemplate belongs to
     feedback: General feedback left by a Researcher for a Student
+    self_rating: The rating a student gives their performance. Should be out of 10
     """
     id = models.UUIDField(unique=True, editable=False, primary_key=True, default=uuid.uuid4)
     completion_date = models.DateTimeField(default=None, null=True)
     student = models.ForeignKey('users.Student', related_name='template_responses', default=0, on_delete=models.CASCADE)
     template = models.ForeignKey('conversation_templates.ConversationTemplate', default=0, related_name='template_responses', on_delete=models.CASCADE)
     assignment = models.ForeignKey('users.Assignment', default=0, related_name='template_responses', on_delete=models.CASCADE)
     feedback = models.CharField(max_length=1000, default=None, null=True, blank=True)
+    self_rating = models.IntegerField(default=0, null=True)
```

Comment:
It looks like the default here is 0 but the default in conversation_end is 3. I think it should be consistent between the two.

Reply:
Can we change the field to PositiveSmallIntegerField? I will change in my PR as well unless doth protest

Reply:
I forgot to change it back to 0. Was 3 for testing

Sure will change